XenAdmin/TestResources/api-version.xml

```diff
@@ -160,5 +160,6 @@
     <call name="secret_set_value" />
     <call name="secret_get_all" />
     <call name="secret_get_all_records" />
+    <call name="vm_call_plugin" />

```

Comment:
This file and callVersions.csv need to be autogenerated: I think you have a bug here that the call is in the wrong server version. But in any case, they are very out of date, so I suggest just ignoring them and not making any changes to them.